ui.frontend/src/view/FormContainer.js

```diff
@@ -185,6 +185,34 @@ class FormContainer {
     getAllFields() {
         return this._fields;
     }
+
+    /**
+       * Updates the active child of the form container.
+       * @param {Object} activeChild - The active child.
+    */
+    updateActiveChild(activeChild) {
+      this.setFocus(activeChild?._activeChild?.id || activeChild?.id);
+    }
+
+    /**
+       * Subscribes to model changes and updates the corresponding properties in the view.
+       * @override
+     */
+    subscribe() {
+        const changeHandlerName = (propName) => `update${propName[0].toUpperCase() + propName.slice(1)}`
```

Comment:
Can we re-use this logic by moving it to utility class via composition ?

Reply:
A new Base class needs to be created. Both FormContainer and FormField classes will inherit this newly created Base class.
can we do this?